lib/beaker/host_prebuilt_steps.rb

```diff
@@ -555,6 +567,10 @@ def set_env host, opts
           host.exec(Command.new("touch #{host[:ssh_env_file]}"))
           #add the constructed env vars to this host
           host.add_env_var('PATH', '$PATH')
+          # FIXME
+          if host['platform'] =~ /openbsd-([^-]+)-(.+)/
+            host.add_env_var('PKG_PATH', "http://ftp.openbsd.org/pub/OpenBSD/#{$1}/packages/#{$2}/")
```

Comment:
This should probably auto-translate x64 -> amd64 (ie. http://ftp.openbsd.org/pub/OpenBSD/5.6/packages/amd64/)